Assignment4/c.paneque-21A280/task06.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#"""Task06.ipynb
+
+#Automatically generated by Colaboratory.
+
+#Original file is located at
+   # https://colab.research.google.com/drive/1TKFH51IowiMiu6TQLp-WiduCnFnxRKkk
+
+#**Task 06: Modifying RDF(s)**
+#"""
+
+!pip install rdflib 
```

Comment:
This won't run in Python. You need to comment out line 12. The same applies for the other tasks